### ISSUE TITLE
kcov seems to loosing trace when functions called `xargs bash -c function`

### ISSUE THREAD

**Issue by vlajos** [SimonKagstrom/kcov#237]
In my script I try to parallelize mass processing of files with this `xargs bash -c bashfunction` solution. 
For example:
https://coveralls.io/builds/15790039/source?filename=misspell_fixer.sh

Is there an easy way to get coverage report about these functions as well?

**Comment by vlajos** [comment]
Thank you for the answer.

> Hmm... I'm not sure. So this starts a new bash shell to execute the function in?

Yes. I did not find a better way to parallelize the execution 

I tried the suggested 2 options, but you were right. They did not make any difference.

**Comment by vlajos** [comment]
@SimonKagstrom I was wondering maybe to generate individual data for all the xargs-forked instances and merge them somehow. Would this be possible? What do you think?

**Comment by SimonKagstrom** [comment]
Yes! That should be possible. kcov has a `--merge` option, where you can produce a merged output from multiple separate runs.

Also, if you run with the same output directory, merging is done automatically and placed under the `[merged]` link in the html output.

**Comment by vlajos** [comment]
Thanks.
I think I managed to implement it...
I had to create a wrapper script, kcov did not like bash as an argument, but it seems to be working now:
https://github.com/vlajos/misspell-fixer/commit/efe0ea657d57826a72813afac5f585bd390ce372
